NAMESPACE

```diff
@@ -37,6 +38,7 @@ importFrom(bnlearn,arcs)
 importFrom(bnlearn,fast.iamb)
 importFrom(dplyr,`%>%`)
 importFrom(dplyr,select)
+importFrom(magrittr,"%>%")
```

Comment:
the `%>%` is already imported via `dplyr`. Is there any specific reason to have import it from `magrittr`?

Reply:
You're right. Sorry I haven't seen this